index.html

```diff
@@ -2247,6 +2091,175 @@
   monitoring service integrity, load balancing, capacity planning, or billing); and,
 * has a contract in place with the [=actor=] which is consistent with the above limitations.
 
+## Recognition {#recognition}
+
+<dfn data-lt="recognize|recognized">Recognition</dfn> is the act of realising that a given [=identity=]
+corresponds to the same [=person=] as another [=identity=] which may have been
+observed either in another [=context=], or in the same [=context=] but at a
+different time.
+
+A [=person=] can be [=recognized=] whether or not their legal identity or
+characteristics of their legal identity are included in the recognition.
+
+### Recognition Types {#recognition-types}
+
+There are several types of [=recognition=] that may take place.
+
+<dfn>Cross-context recognition</dfn> is [=recognition=] between different
+[=contexts=]. It contributes to [=surveillance=], [=correlation=],
+and [=identification=].
+
+[=Cross-context recognition=] is only [=appropriate=] when the person being [=recognized=]
+can reasonably expect recognition to happen, and can control whether it does.
+
+If a person can uses a piece of identifying information in two different
+contexts (e.g. their email or phone number), this *does not* automatically
+mean that they're using the same identity in both contexts. It is
+[=inappropriate=] to [=recognize=] them using that information, unless there's
+some other indication that they intended to use a single identity. It is also
+[=inappropriate=] to seek extra identifying information to help with
+cross-context recognition.
+
+Systems which [=recognize=] people across [=contexts=] need
+to be careful not to apply the principles of one [=context=] in ways that
+violate the principles around use of information acquired in a different
+[=context=]. This is particularly true for [=vulnerable=] people, as
+recognising them in different [=contexts=] may force traits into the open
+that reveal their vulnerability. For example, if you meet your therapist at a
+party, you expect them to have different discussion topics with you than they
+usually would, and possibly even to pretend they don't know you.
+
+<dfn>Cross-site recognition</dfn> is when a [=site=] determines with high
+probability that a visit to the site comes from the same person as another
+visit to a *different* site. In the usual case that the sites are different [=contexts=],
```

Comment:
If we move the high-probability bit up to the main definition of recognition, this becomes:

```suggestion
<dfn>Cross-site recognition</dfn> is [=recognition=] when the identities
are observed on different [=sites=]. In the usual case that the sites are
different [=contexts=],
```